skbio/diversity/alpha/_base.py

```diff
@@ -258,8 +362,14 @@ def esty_ci(counts):
     return f1 / n - z * np.sqrt(W), f1 / n + z * np.sqrt(W)
 
 
+def _faith_pd(counts_by_node, branch_lengths):
+    counts_by_node = np.where(counts_by_node > 0, 1, 0)
+    result = (branch_lengths * counts_by_node).sum()
+    return result
+
+
 @experimental(as_of="0.4.0-dev")
-def faith_pd(counts, otu_ids, tree, validate=True):
+def faith_pd(counts, otu_ids, tree, validate=True,):
```

Comment:
Let's move ``faith_pd`` to a new ``_faith_pd.py`` file, and then rename the test file to mirror that.